projects/ui/src/util/BigNumber.ts

```diff
@@ -11,3 +13,22 @@ export const BN = (v: BigNumber.Value) => new BigNumber(v);
 //     hex: `0x${this.toString(16)}`,
 //   };
 // };
+
+export function normalizeBN(
+  value: BigNumber | undefined | null,
+  _gt?: BigNumber.Value,
+) {
+  return (value && value.gt(_gt || 0) ? value : ZERO_BN);
+}
+
+export function tokenValueToBN(
```

Comment:
So the reason why I've added these is because we use a different BigNumber package in the sdk. I believe in the sdk we use bn.js and in the ui we use bignumber.js.

Reply:
Oh, you're right, missed that, my bad